src/less/success-message/base/success-message.less

```diff
@@ -40,6 +20,25 @@
     padding: 0 9%;
     text-align: center;
     width: 100%;
+
+    .btn--secondary,
+    .fake-btn--secondary {
```

Comment:
Also on the same note, I made my styles match `btn`, should we switch this to do the same for consitancy. Part of my reasoning was if they wanted to switch from `seconday--btn` to something else, there wont be any breaking changes. Also for my case (and I hope the same is here too) having stronger selectors will not cause any issues.